lib/network/ci-external-load-balancer.ts

```diff
@@ -41,11 +42,21 @@ export class JenkinsExternalLoadBalancer {
     const accessPort = props.useSsl ? 443 : 80;
 
     this.listener = this.loadBalancer.addListener('JenkinsListener', {
+      sslPolicy: SslPolicy.RECOMMENDED,
```

Comment:
What is        sslPolicy: SslPolicy.RECOMMENDED?
Can you specifically define it to TLS1-2-2017-01?

Reply:
CDK recommends which works best for our set up. Hence keeping it that way.

Reply:
More details: https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-elasticloadbalancingv2.SslPolicy.html

Reply:
We get approved specifically for 2017 one, please do not use CDK recommended one and use the tls 1 2 2017 01 specifically.

Reply:
Thanks.

Reply:
Changed and tested! Thanks!

Reply:
Nitpik, add a comment to state that ` TLS12 = 'ELBSecurityPolicy-TLS-1-2-2017-01'`

Reply:
You can see that if you click on the enum value from IDE.

Reply:
That is not good enough for code reviewing, but ok.

Reply:
I just want to code to be more clear without additional tools.